lambda_functions/v1/functions/lpa_codes/app/api/code_generator.py

```diff
@@ -104,8 +104,7 @@ def get_codes(database, key=None, code=None):
             query_result = table.query(
                 IndexName="key_index",
                 KeyConditionExpression=Key("lpa").eq(lpa) & Key("actor").eq(actor),
-                FilterExpression=Attr("expiry_date").gt(ttl_cutoff),
-                ProjectionExpression=return_fields,
```

Comment:
I removed this as it did not include the generated_date in the response - is this okay?